### ISSUE TITLE
Create expedition landing page

### ISSUE THREAD

**Issue by jdeck88** [biocodellc/geome-ui#485]
Each expedition, e.g. https://n2t.net/ark:/21547/DLY2  should have a landing page of its own.  This would display expedition specific data properties as returned by the expedition metadata from the REST response.  Currently the expedition redirects end up at the EZID resolver (after being sent to GEOME).

**Comment by albarqmaria** [comment]
We need a new GET request that takes the ARK number before we can implement this in the front end

**Comment by jdeck88** [comment]
# the problem

All ARKs of the form:
ark:/21547/{entity identifier}2* (which have characters following the number 2) are records and resolved like this:

```
https://develop.geome-db.org/record/ark:/21547/lA2Aac1
```

The above URL succesfully resolves to a sample (or whatever entity) landing page.  The ```lA2``` above is the entity identifier.   ```Aac1``` is the suffix. 

If you try to resolve the identifier with out the suffix, like so:

```
https://develop.geome-db.org/record/ark:/21547/lA2
```
you get redirected to the Name-to-thing resolution service at ```https://ezid.cdlib.org/id/ark:/21547/lA2``` Instead we want to redirect all "root-level" identifiers, that is, identifiers that have no suffix to the service running at ```bcids/metadata```.  This API service call looks like the following:

```
https://api.develop.geome-db.org/bcids/metadata/ark:/21547/lA2
```
Note that the ```/record/``` above is the UI endpoint and ```/bcids/metadata``` is the API endpoint, so we need to modify the ```/record/``` UI endpoint to resolve root-level identifiers to ```/bcids/metadata```

# the solution
There are two types of root-level identifiers:  

  1) Entity type identifiers which are Sample, Tissue, Event, etc...
  2) Expedition type identifiers which have an expeditionTitle, expeditionCode, and all of the Entities.  

We need to provide landing pages for both of the above and take care of this in the ```/records``` service.  The entity identifier has a response that contains an ```entityIdentifier``` attribute in the root level of the returned JSON, see for example:

```
https://api.develop.geome-db.org/bcids/metadata/ark:/21547/lA2
```

The expedition identifier has a response that contains an ```expedition``` attribute in the root level of the returned JSON, see for example:

```
https://api.develop.geome-db.org/bcids/metadata/ark:/21547/AEq2
```

Both of these landing pages (Entities and Expeditions) will be resolved through calls to the ```/record/``` URL (as served by angular).    We just need to unpack the JSON in the responses depending on the type of data. In the case of a suffix point to the landing page for the Sample, Tissue, or Event as we are doing now.  In the case of an expedition we point to the expedition landing page and in the case of an Entity, just say "Sample" and point to the parent expedition.

**Comment by albarqmaria** [comment]
Expedition landing page has: expedition code, title, date of create, date of modification, visibility,
and a bullet list of all the entities in the expedition. When you click on the entities GUID, you get an entity landing page. 
We want to loop through expedition metadata on the display page, because there are custom fields.

![Screen Shot 2020-07-31 at 8 37 20 AM](https://user-images.githubusercontent.com/35155534/89051409-0deca380-d309-11ea-8c39-3e3c1e029a97.png)
url will look like: geome-db.org/bcids/metadata/ark:/identifiernumber

Entity landing page will include: identifier, conceptAlias, parentExpedition. The idea is to make a parent page, that explains its a root level identifier page, maybe include a link to query all the samples, gives a path to point to the expedition Metadata as well as project metadata.

**Comment by albarqmaria** [comment]
![Screen Shot 2020-08-14 at 8 13 03 AM](https://user-images.githubusercontent.com/35155534/90264452-fb449500-de05-11ea-90a6-ed540cc3af53.png)

![Screen Shot 2020-08-14 at 8 13 16 AM](https://user-images.githubusercontent.com/35155534/90264478-03043980-de06-11ea-901b-8712a06f3718.png)

Here's my mock-ups on the designs for these two views, please let me know what changes  are needed

**Comment by jdeck88** [comment]
I would go for a more sensible title for the expedition.  Also use the fully qualified ARK
So:
**GT-seq trials (1001)**
Identifier: https://n2t.net/ark:/21547/xxxxxx
rest of metadata.....

For the Entity, put i would make the title like:

**'Pomo Moorea' Events**
... then the rest of the metadata (be sure to use fully qualified arks).  
... also make a link to expedition title using like <a href="link URL">Expedition Title</a?

**Comment by albarqmaria** [comment]
OK, going to submit a PR for this.
I ended up modifying the styling a bit, to make the page look more like
the other records views. 
Also, I used the identifier as a link to the parent expedition, rather that using Expedition Title.
Let me know if this works or not, we can continue discussion in the PR.

**Comment by albarqmaria** [comment]
this was addressed by PR #523
